pkg/pillar/cmd/zedkube/zedkube.go

```diff
@@ -531,6 +542,7 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 			zedkubeCtx.collectAppLogs()
 			zedkubeCtx.checkAppsStatus()
 			zedkubeCtx.collectKubeStats()
+			zedkubeCtx.collectKubeSvcs()
```

Comment:
Wouldn't it be more efficient to watch on Service and Ingress object changes rather than re-running and re-publishing KubeUserServices every 30 seconds?
I also wonder how much device logs are generated and sent to the controller from this periodic action.